lib/index.js

```diff
@@ -116,17 +117,18 @@ function addDataToSwaggerObject(swaggerObject, data) {
           }
           break;
         }
-        case 'tag': {
+        case 'tags': {
```

Comment:
Add a case for 'tag', don't break and add a `console.warn('tag will be deprecated in v1.0.0, please use tags as it aligns with the swagger v2.0 spec')` then set `propertyName = 'tags';`